shipment_advice/models/stock_move.py

```diff
@@ -23,3 +24,9 @@ def _prepare_merge_moves_distinct_fields(self):
         # Avoid having stock move assign to different shipment merged together
         res.append("shipment_advice_id")
         return res
+
+    def _prepare_move_split_vals(self, qty):
+        vals = super()._prepare_move_split_vals(qty)
+        if self.env.context.get("shipment_advice__propagate_on_split"):
```

Comment:
@TDu @jbaudoux Why this key of context ?

Shouldn't it be possible to use the stock rule that has created the move and a property on it to do the same ?

Reply:
Just as Information, we discussed this topic some weeks ago and there was no scenario where the id should not be copied. All of the cases in shopfloor as example would need the value. 
So i think we should close this one.

Reply:
It seems that this PR is not used on customer projects, I would say it can be closed.
Although I think the `copy=False` makes some sense.